okonf/facts/systemd.py

```diff
@@ -59,3 +59,21 @@ async def enforce(self, host: Executor) -> bool:
         async with host.lock(f"systemctl-service-{self.name}"):
             await host.check_output(f"sudo systemctl restart {self.name}")
         return True
+
+
+class ServiceEnabled(Fact):
+    """Ensure that a Systemd is enabled"""
+
+    is_stateless = True
```

Comment:
`is_stateless` is reserved for facts that are stateless: they do not change the state of the system. Having a service started or enabled changes the state, while having it restarted does not change the state (it was running both before and after).